### ISSUE TITLE
Null Pointer Exception if only member of household is marked Deceased and opportunity is added for that contact

### ISSUE THREAD

**Issue by judisohn** [SalesforceFoundation/Cumulus#2060]
Thread: https://powerofus.force.com/0D58000002eBDjh

**Comment by judisohn** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x80000003abRWAAY" target="_blank">W-010058</a>

**Comment by johnfine** [comment]
In addition to the usage scenario I described in the POU thread, this error also occurs when performing a mass update operation (not only a mass insert operation).  I was updating Opp.npsp__primarycontact__c as part of an in-place migration from GWBASE to NPSP3, and because of 150 Deceased Contacts, I got 21000 failures out of 25000 records.  My workaround was to uncheck COntact.npsp__deceased__c for all Contacts, perform the Opp update, then check Contact.npsp__deceased again for those Contacts.
